hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapOperation.java

```diff
@@ -56,6 +56,7 @@
      * otherwise in active-active scenarios infinite loop of event forwarding can be seen.
      */
     protected boolean disableWanReplicationEvent;
+    private boolean canPublishWanEvent;
```

Comment:
The field should be marked as transient to stay consistent with the other not serialized fields and to show the intention. Or we should keep the `canPublishWanEvent()` method call below and remove the field.

Reply:
done.